codecov.yml

```diff
@@ -1,19 +0,0 @@
-codecov:
```

Comment:
removed because it was duplicated, the correct one has the extension `codecov.yaml`